coursier.bzl

```diff
@@ -1048,6 +1064,8 @@ pinned_coursier_fetch = repository_rule(
         "jetify_include_list": attr.string_list(doc = "List of artifacts that need to be jetified in `groupId:artifactId` format. By default all artifacts are jetified if `jetify` is set to True.", default = JETIFY_INCLUDE_LIST_JETIFY_ALL),
         "additional_netrc_lines": attr.string_list(doc = "Additional lines prepended to the netrc file used by `http_file` (with `maven_install_json` only).", default = []),
         "fail_if_repin_required": attr.bool(doc = "Whether to fail the build if the maven_artifact inputs have changed but the lock file has not been repinned.", default = False),
+        "starlark_aar_import": attr.bool(default = False, doc = "Whether to use the native or Starlark version of aar_import"),
+        "aar_import_bzl_label": attr.string(default = DEFAULT_AAR_IMPORT_LABEL, doc = "The label (as a string) to use to import aar_import from"),
```

Comment:
I don't think we need an attribute for this, right? This can be an implementation detail of the other attribute within the function bodies of coursier_fetch and pinned_coursier_fetch.

Reply:
So the idea here is that the generated build file needs to have the label to the bzl file for aar_import, and the sample in the rules_android readme gives an http_archive with the name "build_bazel_rules_android", but repos could have a different name (e.g. just "rules_android"). Also, the label within the rules_android repo may differ between versions of the android rules. The current code requires "//android:rules.bzl", but the pre-alpha code requires "//rules:rules.bzl". So this provides some flexibility during migrations.